### DISCUSSION TITLE
HERBIE in a weather website back-end

### DISCUSSION THREAD

**Discussion by Fournogo** [blaylockbk/Herbie#229] [Show and tell]
This is probably way outside of what you intended for HERBIE but I did it anyway lol

I made this weather website: https://scrump.cfd/weather.html

If you scroll down toward the bottom, above the sounding and below the cloud cover forecast, there's a model data window. This is a hobby project (mainly for my own enjoyment but anyone is welcome to use it) so it's not really mobile optimized. I recommend checking it out on a desktop/laptop and maximizing the model data window. Then select "browser" and you'll be able to switch from HRRR to GFS data and look at a couple products. I'll be expanding the set of products available and adding some more useful features in the model browser soon, it's still a work in progress! 

The code for the project is available here: https://github.com/Fournogo/SCRUMP.CFD

In the "Python" section under "state_plots.py" or "national_plots.py" you can see where HERBIE is being used to pull all the model data for these animations and then generating the images associated with them. I'm not the best programmer so disregard the sloppiness in the code.. I'll tighten it up in time haha. I'm about at the point where I really need to start optimizing

Either way, hope you enjoy a stroll through the website and thank you for all your hard work on HERBIE!

**Comment by cmo84** [comment]
Where do I sign to petition to replace noaa.gov with this site?

**Reply by Fournogo** [reply]
Write your local congressman! haha

**Comment by eroesler** [comment]
🤣

**Comment by blaylockbk** [comment]
Hey that's pretty cool! Thanks for sharing. It's neat to see how Herbie is being used.

Fun fact, one of Herbie's original use cases was to dynamically create web images for fire weather applications. This page (amazingly) still works and uses a very early version of Herbie https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_custom.html

**Reply by Fournogo** [reply]
That's awesome! I may have to take some inspiration from how you guys do things once I start tightening my website. Thank you again!
